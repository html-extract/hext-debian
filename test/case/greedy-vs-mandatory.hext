<+p class:greedy1 />
<h1 class:h />
<+p class:greedy2 />
<p class:p />
