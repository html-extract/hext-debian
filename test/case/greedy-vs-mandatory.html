<p class="greedy1"></p>
<h1 class="h1"></h1>
<p class="greedy2"></p>
<p class="p1"></p>

<p class="greedy1"></p>
<p class="greedy1"></p>
<h1 class="h1"></h1>
<p class="greedy2"></p>
<p class="p1"></p>
<p class="p2"></p>

