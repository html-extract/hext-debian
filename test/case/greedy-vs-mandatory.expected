{"greedy1":"greedy1","greedy2":"greedy2","h":"h1","p":"p1"}
{"greedy1":["greedy1","greedy1"],"greedy2":"greedy2","h":"h1","p":"p1"}
